Sources/Orbit/Components/Timeline.swift

```diff
@@ -99,6 +99,7 @@ struct TimelinePreviews: PreviewProvider {
             TimelineStep(
                 "Arrive",
                 sublabel: "May 4, 11:49",
+                style: .default,
```

Comment:
Lets add custom content to some of these in the `Timeline` preview

Reply:
Added with modifiers. Without modifiers it wont work at least for now.